src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/service/SendNotificationService.java

```diff
@@ -216,13 +221,30 @@ public void sendNotifyEmails(CaseDetails caseDetails) {
             return;
         }
 
-        String claimantEmailAddress = caseData.getClaimantType().getClaimantEmailAddress();
+        // Send notification to the claimant Only
         String caseId = caseDetails.getCaseId();
         if (!RESPONDENT_ONLY.equals(caseData.getSendNotificationNotify())) {
-            emailService.sendEmail(claimantSendNotificationTemplateId, claimantEmailAddress,
-                    buildPersonalisation(caseDetails, emailService.getCitizenCaseLink(caseId)));
+            // If represented, send notification to claimant representative Only
+            Map<String, String> personalisation;
+            if (isRepresentedClaimantWithMyHmctsCase(caseDetails.getCaseData())) {
+                personalisation = NotificationHelper.buildMapForClaimantRepresentative(caseDetails.getCaseData());
+                personalisation.put(LINK_TO_EXUI,
+                        emailService.getClaimantRepExuiCaseNotificationsLink(caseDetails.getCaseId()));
+                if (!isNullOrEmpty(personalisation.get(EMAIL_ADDRESS))) {
+                    emailService.sendEmail(claimantRepSendNotificationTemplateId, personalisation.get(EMAIL_ADDRESS),
+                            personalisation);
+                }
+            } else {
+                // If not represented, send notification to the claimant Only
+                String claimantEmailAddress = caseData.getClaimantType().getClaimantEmailAddress();
+                if (!isNullOrEmpty(claimantEmailAddress)) {
+                    emailService.sendEmail(claimantSendNotificationTemplateId, claimantEmailAddress,
+                            buildPersonalisation(caseDetails, emailService.getCitizenCaseLink(caseId)));
+                }
```

Comment:
I think should maybe be wrapped in an else if instead where we check to see if the claimant is a system user as well. There are claims on prod where claimants do not have access to citizen hub so we need to make sure we aren't sending them emails to something they can't access

Reply:
I add fix to cover that scenario.